symbolic-symcache/src/writer.rs

```diff
@@ -194,7 +192,8 @@ impl<W> SymCacheWriter<W>
 where
     W: Write + Seek,
 {
-    /// Converts an entire object into a SymCache.
+    /// Converts an entire object (an instance of a type that implements [`ObjectLike`](ObjectLike))
```

Comment:
This should resolve automatically:

```suggestion
    /// Converts an entire object (an instance of a type that implements [`ObjectLike`])
```